my-site.rb

```diff
@@ -1,8 +1,19 @@
 require "sinatra"
 
 class MySite < Sinatra::Base
-
   get "/" do
-
+    erb :index
+  end
```

Comment:
For readability, I would add one line of whitespace between each verb/route combo, so after line 6, 9, 12, etc.